dev-haskell/http-client/http-client-0.7.10.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
```

Comment:
I'd say work through conflicts and pull in your changes, as it allows us to use the rest of the test-suite that doesn't need the network.